kb(Tooltip): kb for changing callout position (#2062)
telerik/blazor-docs: knowledge-base/tooltip-callout-position.md

---
title: Hide the Tooltip Callout or Change its Position
description: Learn how to customize the appearance and behavior of the Tooltip small arrow, including adjusting the callout position and hiding it when necessary.
type: how-to
page_title: Hide the Tooltip Callout or Change its Position
slug: tooltip-kb-callout-position
tags: tooltip, styling, callout
ticketid: 1628953, 1565205
res_type: kb
---

## Environment

<table>
    <tbody>
        <tr>
            <td>Product</td>
            <td>Tooltip for Blazor</td>
        </tr>
    </tbody>
</table>

## Description
This Knowledge Base article answers the following questions:

> * How to customize the Tooltip callout ?

Reviewer: You don't need a space between the last word and the ending punctuation in a sentence. Remove for all bullet points. 
```suggestion
* How to customize the Tooltip callout?
```

> * How to change the positon of the Tooltip callout ?

Reviewer: ```suggestion
* How to change the position of the Tooltip callout?
```

* How to modify the position of the small arrow of a Tooltip ?
* How to hide the Tooltip callout ?

## Solution

> 1. Set a custom CSS class to the Tooltip through the `Class` parameter. This configuration will allow you to target specific Tooltip instances.

Reviewer: Present Simple is usually preferable over other tenses unless it modifies the meaning of the sentence. See [here](https://docs.telerik.com/style-guide/basic-rules-and-guidelines#use-simple-tenses).

```suggestion
1. Set a custom CSS class to the Tooltip through the `Class` parameter. This configuration allows you to target specific Tooltip instances.
```

> 2. Use the defiend class to [Override the theme styles](https://docs.telerik.com/blazor-ui/styling-and-themes/override-theme-styles).

Reviewer: ```suggestion
2. Use the defined class to [Override the theme styles](https://docs.telerik.com/blazor-ui/styling-and-themes/override-theme-styles).
```

3. Customize the callout (small arrow) position by using this CSS combinator `.your-class .k-callout`, to suit your specific requirements.

> 3. Set a value of the TargetSelector parameter of the Tooltip.

Reviewer: ```suggestion
3. Set a value for the `TargetSelector` parameter of the Tooltip.
```

---
title: Hide the Tooltip Callout or Change Its Position
description: Learn how to customize the appearance and behavior of the Tooltip small arrow, including adjusting the callout position and hiding it when necessary.
type: how-to
page_title: Hide the Tooltip Callout or Change Its Position
slug: tooltip-kb-callout-position
tags: tooltip, styling, callout
ticketid: 1628953, 1565205
res_type: kb
---

## Environment

<table>
    <tbody>
        <tr>
            <td>Product</td>
            <td>Tooltip for Blazor</td>
        </tr>
    </tbody>
</table>

## Description
This Knowledge Base article answers the following questions:

* How to customize the Tooltip callout?
* How to change the position of the Tooltip callout?
* How to modify the position of the small arrow of a Tooltip?
* How to hide the Tooltip callout?

## Solution

1. Set a custom CSS class to the Tooltip through the `Class` parameter. This configuration allows you to target specific Tooltip instances.

> 2. Use the defined class to [Override the theme styles]({%slug themes-override%}).

Reviewer: ```suggestion
2. Use the defined class to [override the theme styles]({%slug themes-override%}).
```
